src/raster/array.jl

```diff
@@ -1,89 +1,133 @@
+import DiskArrays: eachchunk, GridChunks
```

Comment:
I think 
```
import DiskArrays

...

function DiskArrays.eachchunk(...)

function DiskArrays.GridChunks(...)
```

might make it clearer that multi-dispatch is happening below.

Reply:
Yes, that makes sense,